src/hooks/usePortalContainer/README.md

```diff
@@ -0,0 +1,17 @@
+<!--GITHUB_BLOCK-->
+
+# usePortalContainer
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {usePortalContainer} from '@gravity-ui/uikit';
+```
+
+The `usePortalContainer` hook returns portal container element in the current scope
+
+## Result
+
+| Name      | Description                  |         Type         |
+| :-------- | :--------------------------- | :------------------: |
+| container | Container element for portal | `HTMLElement - null` |
```

Comment:
This table is not correct for result. There is no `container` name in result, result itself is the container. Let's describe like this:

```
## Result

Container element for portal. `HTMLElement | null`
```